content-src/experiments/lockbox.yaml

```diff
@@ -0,0 +1,129 @@
+    copy: >
+      Access username and password information for all the accounts you save to Firefox.
+
+  -
+    image: /static/images/experiments/lockbox/details/sync.png
+    copy: >
+      Manage accounts in Firefox and sync changes automatically to Firefox Lockbox.
+
+  -
+    image: /static/images/experiments/lockbox/details/sort-filter.png
+    copy: >
+      Sort and filter your accounts to quickly locate the one you need.
+
+contributors:
+  -
+    display_name: 'Sandy Sage'
+    title: 'Sr. Product Manager'
+    avatar: /static/images/experiments/lockbox/avatars/ssage.png
+  -
+    display_name: 'Devin Reams'
+    title: 'Engineering Manager'
+    avatar: /static/images/experiments/lockbox/avatars/dreams.png
+  -
+    display_name: 'Ryan Gaddis'
+    title: 'Sr. Interaction Designer'
+    avatar: /static/images/experiments/lockbox/avatars/rgaddis.png
+  -
+    display_name: 'Matt Miller'
+    title: 'Engineering Architect'
+    avatar: /static/images/experiments/lockbox/avatars/mmiller.png
+  -
+    display_name: 'Sasha Heinen'
+    title: 'Mobile Engineer'
+    avatar: /static/images/experiments/lockbox/avatars/sheinen.png
+  -
+    display_name: 'Jim Porter'
+    title: 'Sr. Engineer'
+    avatar: /static/images/experiments/lockbox/avatars/anon.png
+  -
+    display_name: 'James Hugman'
+    title: 'Mobile Engineer'
+    avatar: /static/images/experiments/lockbox/avatars/jhugman.png
+  -
+    display_name: 'Leif Oines'
+    title: 'Product Data Scientist'
+    avatar: /static/images/experiments/lockbox/avatars/loines.png
+  -
+    display_name: 'Heather McGaw'
+    title: 'Sr. UX Researcher'
+    avatar: /static/images/experiments/lockbox/avatars/hmcgaw.png
+  -
+    display_name: 'Michelle Heubusch'
+    title: 'Content Strategy Lead'
+    avatar: /static/images/experiments/lockbox/avatars/mheubusch.png
+  -
+    display_name: 'Nick Brandt'
+    title: 'Sr. UX Designer'
+    avatar: /static/images/experiments/lockbox/avatars/nbrandt.png
+  -
+    display_name: 'Matt Brandt'
+    title: 'Sr. Test Engineer'
+    avatar: /static/images/experiments/lockbox/avatars/mbrandt.png
+  -
+    display_name: 'Isabel Rios'
+    title: 'Mobile QA Engineer'
+    avatar: /static/images/experiments/lockbox/avatars/irios.png
+  -
+    display_name: 'Simion Basca'
+    title: 'Mobile QA Engineer'
+    avatar: /static/images/experiments/lockbox/avatars/sbasca.png
+  -
+    display_name: 'Catalin Sasca'
+    title: 'Mobile QA Engineer'
+    avatar: /static/images/experiments/lockbox/avatars/anon.png
+  -
+    display_name: 'Peter deHaan'
+    title: 'Firefox QA'
+    avatar: /static/images/experiments/lockbox/avatars/pdehaan.png
+  -
+    display_name: 'Joe Gasiorek'
+    title: 'Community Contributor'
+    avatar: /static/images/experiments/lockbox/avatars/jgasiorek.png
+  -
+    display_name: 'Paul Nyondo'
+    title: 'Community Contributor'
+    avatar: /static/images/experiments/lockbox/avatars/pnyondo.png
+
+measurements:
+  - >
+    REMOVE ME
```

Comment:
I haven't looked into TestPilot schemas lately to see if `measurements` is required, but I'll just call this out because it is literally telling me to remove it.